pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -1,24 +1,43 @@
 # coding=utf-8
+# pylint:disable=too-many-lines
 """Tests that perform actions over RPM modular repositories."""
 import gzip
 import io
 import unittest
+
+from collections import defaultdict
```

Comment:
@bherrin3.

Reply:
Thanks. Using ``isort`` to get the lib order correct (reference) for future refactors. Thanks for the head's up.